src/main/java/adhd/diary/auth/CustomOAuth2User.java

```diff
@@ -0,0 +1,36 @@
+package adhd.diary.auth;
+
+import adhd.diary.member.domain.Role;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.oauth2.core.user.DefaultOAuth2User;
+
+import java.util.Collection;
+import java.util.Map;
+
+public class CustomOAuth2User extends DefaultOAuth2User {
+
+    private String socialId;
+    private String email;
+    private Role role;
+
+    public CustomOAuth2User(Collection<? extends GrantedAuthority> authorities,
+                            Map<String, Object> attributes, String nameAttributeKey,
+                            String email, Role role, String socialId) {
```

Comment:
```suggestion
    public CustomOAuth2User(Collection<? extends GrantedAuthority> authorities,
                            Map<String, Object> attributes, 
                            String nameAttributeKey,
                            String email, 
                            Role role, 
                            String socialId) {
```
파라미터 3개 이상 오면 하나씩 작성하기로 했으니 수정 부탁드려요!